src/msgwindow.c

```diff
@@ -315,11 +317,15 @@ static const GdkColor *get_color(gint msg_color)
 
 
 /**
- *  Adds a new message in the compiler tab treeview in the messages window.
+ * Adds a formatted message in the compiler tab treeview in the messages window.
+ *
+ * @param msg_color A color to be used for the text. It must be an element of #MsgColors.
+ * @param format    @c printf()-style format string.
+ * @param ...       Arguments for the @c format string.
  *
- *  @param msg_color A color to be used for the text. It must be an element of #MsgColors.
- *  @param format @c printf()-style format string.
- *  @param ... Arguments for the @c format string.
+ * @see msgwin_compiler_add_string()
+ *
+ * @since 0.15
```

Comment:
Is that correct?  Looks to me it's new in 921f010ffc38aaadf7118161b2f85df58e7bfc5c which seems to be [0.16](https://github.com/geany/geany/blob/921f010ffc38aaadf7118161b2f85df58e7bfc5c/configure.in#L5)
It doesn't matter much at all, but wrong info isn't nice.

Reply:
Yes, you are right. I didn't spot 0.15 when going backwards in the log starting from that commit. Next time I shall just ask `git describe`

Reply:
I know what happened. I was looking at c151beff which had added these function under a different name (_add_fmt()). For 0.16.0 they were renamed to the current name. So yes, 0.16 is more accurate.